django/utils/timezone.py

```diff
@@ -295,13 +295,15 @@ def template_localtime(value, use_tz=None):
 
 # Utilities
 
-def localtime(value, timezone=None):
+def localtime(value=None, timezone=None):
```

Comment:
Looking at the parallels between `localtime` and `localdate` again, I wonder how `localtime` ought to behave when `settings.USE_TZ` is `False`. Seems like it ought to just return `datetime.datetime.now()` (that is, the naive current local time) when `value=None`. I think this could easily be accomplished by just calling `now()` instead of the new `_now_utc()` helper.

When `value` is set and `USE_TZ=False`, I guess perhaps `localtime` should just have no effect? Or just raise an error? But any change here would be backwards-incompatible, so perhaps not worth it.